region/brinstar/blue/Blue Brinstar Boulder Room.json

```diff
@@ -874,10 +874,7 @@
       "requires": [
         "canBombJumpWaterEscape",
         "canJumpIntoIBJ",
-        {"or": [
-          "canTrickyCarryFlashSuit",
-          {"noFlashSuit": {}}
-        ]}
+        "h_trickyToCarryFlashSuit"
```

Comment:
does this one need to remain? I dont know if it is harder than a normal bomb jump water escape.

Reply:
It's more complicated than some other water escapes, because of the tide and because of having to go into an IBJ (not just a single bomb boost). But the difficulty doesn't really have anything to do with the flash suit, nor does it seem Insane level, so probably best to remove it, yeah.